draft-ietf-mls-protocol.md

```diff
@@ -4921,6 +4918,41 @@ be securely removed from a group. It also allows a member to rotate their
 keypair such that the old private key can no longer be used to decrypt new
 messages.
 
+## Confidentiality of Sender Data
+
+The MLSCiphertext framing encrypts "sender data" that identifies which group
+member sent an encrypted message, as described in {{sender-data-encryption}}.
+As with the QUIC header protection scheme {{?RFC9001, Section 5.4}}, this scheme
+is an instance of the HN1 construction analyzed in {{NAN}}.  A sample of the
+ciphertext is combined with a `sender_data_secret` to derive a key and nonce
+that are used for AEAD encryption of the sender data.
+
+``` pseudocode
+(key, nonce) = PRF(sender_data_secret, sample)
+encrypted_sender_data =
+  AEAD.Seal(key, nonce, sender_data_aad, sender_data)
+```
+
+The only difference between this construction and HN1 as described in {{NAN}} is
+that it uses authenticated encryption instead of unauthenticated encryption.
```

Comment:
Plus we don't actually use the ciphertext as nonce, but instead just use it as context in the key/nonce derivation to ensure that the nonce doesn't repeat.